spec/scripts/spec.sh

```diff
@@ -43,4 +43,4 @@ RUBY_VERSION=${VERSION} docker-compose run \
   /bin/bash -c "\
     gem install rake && \
     bundle install && \
-    timeout -s9 5m bundle exec rspec -f progress -f JUnit -o spec/ruby-agent-junit.xml ${TEST}"
+    timeout -s9 5m bin/run-tests ${TEST}"
```

Comment:
I do not see a reason to split the script, it calls another script that is in the folder bin, it is weird

Reply:
Did you see the script?

Reply:
It needs to run _one_ command when passed a `$1` and multiple if not.

Reply:
we can use a function